source/agora/node/FullNode.d

```diff
@@ -500,6 +500,7 @@ public class FullNode : API
         {
             log.error("Error sending updated block headers:{}", e);
         }
+        this.network.getEnrollments(this.enroll_man, this.utxo_set.getUTXOFinder());
```

Comment:
I think it should be its own task, so that it doesn't starve because of block catchup

Reply:
Will modify it.